packages/nimble-components/src/patterns/tabs/testing/tabsbase.pageobject.ts

```diff
@@ -0,0 +1,112 @@
+import type { Button } from '../../../button';
```

Comment:
nit: can you name this file `tabs-base.pageobject.ts`. File names are supposed to be kebab-case but realizing that was overlooked in `pageobject`.